style.css

```diff
@@ -2,12 +2,22 @@
 
 /* melikeaksoy/OC-22-implement-the-footer start */
 
+.newsletter-section {
+  margin-top: 80px;
+  position: relative;
+  z-index: 10;
+  margin-bottom: -90px;
```

Comment:
margin'lerin yuzdelerinin beklendigi gibi calismamasinin sebebi margin yuzdelerini elementin parent'ina gore hesapliyor. O yuzden body'ye nazaran aliyor margin-degerini. 
Bunu cozmenin en temiz yontemi, newsletter section'i ni footer altina alip, 

`  position: absolute;
  top: 0;
  z-index: 10;
  transform: translateY(-50%);`
  degerleriyle oynatmak. 
  
  Ayrica bunu da sizden ekstra rica edecegim, buraya ayrica 
  
  `  width: 90%;
 `
 verelim ki ekran kuculdugunde gorebilelim onu unutmusuz newsletter'i merge ederken.

Reply:
transform translate'in margin'den farki, yuzde degerlerini elementin kendi boyutuna gore hesaplamasinin yaninda, document akisinda elementin kendi yerini reserve ediyor. Yani margin degeriyle bir elementi oynattigimda eleman onceki yerinde bir bosluk birakmiyor ama translate'te ise sadece gorsel olarak oynatiyor, elemanin eski yerinde bosluk kaliyor. 
position:absolute hem onun icin hem de footer'in akisini bozmasin diye kullandim. Tek problemimiz ayrica diger elementlerle boslugunu ayarlarken biraz el ayari yapacagiz.